esdt-safe/src/to_sovereign/events.rs

```diff
@@ -38,7 +36,7 @@ pub trait EventsModule {
     fn deposit_event(
         &self,
         #[indexed] dest_address: &ManagedAddress,
-        #[indexed] tokens: &PaymentsVec<Self::Api>,
+        #[indexed] tokens: &MultiValueEncoded<MultiValue3<TokenIdentifier, u64, BigUint>>,
```

Comment:
Don't use MultiValue types for events. It's bad practice, and doesn't do what you expect it to. Revert to PaymentsVec.

Reply:
@dorin-iancu can you fix this, as you work in this repo ?